subsys/net/ip/ipv6.c

```diff
@@ -65,7 +65,7 @@ static void nd_reachable_timeout(struct k_work *work);
 
 #define MAX_MULTICAST_SOLICIT 3
 #define MAX_UNICAST_SOLICIT   3
-#define DELAY_FIRST_PROBE_TIME (5 * MSEC_PER_SEC) /* RFC 4861 ch 10 */
+#define DELAY_FIRST_PROBE_TIME K_SECONDS(5) /* RFC 4861 ch 10 */
 #define RETRANS_TIMER 1000 /* in ms, RFC 4861 ch 10 */
```

Comment:
We have more "raw" values like this, IMHO makes sense to convert them within the scope of this PR too.